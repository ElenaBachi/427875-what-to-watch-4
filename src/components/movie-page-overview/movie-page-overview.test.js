import React from "react";
import renderer from "react-test-renderer";
import MoviePageOverview from "./movie-page-overview.jsx";

const mock = {
  activeilm: {
    id: 99,
    title: `title`,
    genre: `genre`,
    year: 2020,
    img: `img.jpg`,
    poster: `poster.jpg`,
    cover: `cover.jpg`,
    videoSrc: `video-link`,
    previewVideoSrc: `preview-video-link`,
    description: `description`,
    score: 9,
    count: 200,
    director: `director`,
    actorList: [`Actor1`, `Actor2`, `Actor3`],
    runTime: 100,
  },
};

it(`MoviePageOverview should render correctly`, () => {
  const tree = renderer.create(
      <MoviePageOverview
        activeFilm={mock.activeilm}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
