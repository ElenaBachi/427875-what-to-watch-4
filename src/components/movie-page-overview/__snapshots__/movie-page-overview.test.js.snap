// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageOverview should render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        200
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    description
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Actor1
        Actor2
        Actor3
      </strong>
    </p>
  </div>,
]
`;
